src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -63,6 +63,7 @@
   --pf-c-about-modal-box__header--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
   --pf-c-about-modal-box__header--PaddingBottom: var(--pf-global--spacer--sm);
   --pf-c-about-modal-box__header--PaddingLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  --pf-c-about-modal-box__header--PaddingTop: var(--pf-c-about-modal-box--PaddingTop);
```

Comment:
I think you should just delete this here and where it's referenced. It's never used, so I don't see why we would need to account for it in a `min-height`